Alcatraz/Controllers/ATZPluginWindowController.m

```diff
@@ -255,12 +255,22 @@ - (void)displayScreenshotForPackage:(ATZPackage *)package {
     self.previewPanel.title = package.name;
 
     [self.tableViewDelegate fetchAndCacheImageForPackage:package progress:NULL completion:^(NSImage *image) {
-        [self displayImage:image withTitle:package.name];
+        [self displayImage:image package:package];
     }];
 }
 
-- (void)displayImage:(NSImage *)image withTitle:(NSString*)title {
+- (void)displayImage:(NSImage *)image package:(ATZPackage*)package {
```

Comment:
I'd rename that `displayImage:forPackage:`. This is closest to natural english, and more consistent with `fetchAndCacheImageForPackage:progress:completion:`.